src/SFA.DAS.TrackProgress/DTOs/KsbProgress.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.TrackProgress.DTOs;
+
+public class KsbProgress
+{
+    public long ApprovalId { get; set; }
+    public long ApprovalContinuationId { get; set; }
+    public long ApprenticeshipId { get; set; }
+    public ProgressItem[] Ksbs { get; set; } = null!;
```

Comment:
I'm a little confused what an ApprovalId is. We have the ApprenticeshipID just below, which I would guess would be commitments apprenticeshipId

Reply:
ApprovalID is the `Apprenticeship.Id` in Approvals.  I didn't want to use "apprenticeship" in the name - that is confusing as there are so many ways of representing an apprenticeship.  In reality, it is the identifier of an approval between the employer/provider, so that's the name I used.

We are in the domain of tracking progress here, and our concept of an apprenticeship spans many approvals, therefore it makes sense for our ubiquitous language to represent our view of the world.

The `ApprenticeshipId` we have here is a placeholder for a concept that Roy and payments are working on to create a single ID for an apprenticeship regardless of changes to employer / provider.  This is essentially the same concept that we created in CMAD, however CMAD doesn't actually create the Apprenticeship ID until an apprentice creates their account.